specs/light_client/merkle_proofs.md

```diff
@@ -6,20 +6,53 @@
 <!-- TOC -->
 
 - [Merkle proof formats](#merkle-proof-formats)
-  - [Table of contents](#table-of-contents)
-  - [Generalized Merkle tree index](#generalized-merkle-tree-index)
-  - [SSZ object to index](#ssz-object-to-index)
-    - [Helpers for generalized indices](#helpers-for-generalized-indices)
-      - [`concat_generalized_indices`](#concat_generalized_indices)
-      - [`get_generalized_index_length`](#get_generalized_index_length)
-      - [`get_generalized_index_bit`](#get_generalized_index_bit)
-      - [`generalized_index_sibling`](#generalized_index_sibling)
-      - [`generalized_index_child`](#generalized_index_child)
-      - [`generalized_index_parent`](#generalized_index_parent)
-  - [Merkle multiproofs](#merkle-multiproofs)
+    - [Table of contents](#table-of-contents)
+    - [Custom types](#custom-types)
+    - [Helper functions](#helper-functions)
+    - [Generalized Merkle tree index](#generalized-merkle-tree-index)
+    - [SSZ object to index](#ssz-object-to-index)
+        - [Helpers for generalized indices](#helpers-for-generalized-indices)
+            - [`concat_generalized_indices`](#concat_generalized_indices)
+            - [`get_generalized_index_length`](#get_generalized_index_length)
+            - [`get_generalized_index_bit`](#get_generalized_index_bit)
+            - [`generalized_index_sibling`](#generalized_index_sibling)
+            - [`generalized_index_child`](#generalized_index_child)
+            - [`generalized_index_parent`](#generalized_index_parent)
+    - [Merkle multiproofs](#merkle-multiproofs)
 
 <!-- /TOC -->
 
+## Custom types
+
+We define the following Python custom types for type hinting and readability:
+
+| - | - | - |
+| `GeneralizedIndex` | `uint64` | the index of a node in a binary Merkle tree |
```

Comment:
A GeneralizedIndex is NOT necessarily a uint64. It could be bigger. My idea was to represent it in the python executable as a bigint, but then clarify that representing it as a bitfield/bitlist is ok too.

Reply:
Fixed in https://github.com/ethereum/eth2.0-specs/pull/1363/commits/bbaa238742a93e2aa0524baaf6fd9049d2943d59.